Frameworks/CoreFoundation/Locale.subproj/CFCalendar.c

```diff
@@ -746,6 +746,12 @@ static CFRange __CFCalendarGetRangeOfUnit2(CFCalendarRef calendar, CFCalendarUni
     // WINOBJC: Add cases for weekOf units. ICU will handle them below.
     case kCFCalendarUnitWeekOfYear:
     case kCFCalendarUnitWeekOfMonth:
+    // WINOBJC: Asking for the range of WeekOfMonth in Year always returns 6.
```

Comment:
>Asking for the range of WeekOfMonth in Year always returns 6. [](start = 16, length = 61)

Why do we need a special case here? The logic in calculate: looks sound, as far as I can tell.